kfs-core/src/main/resources/edu/arizona/kfs/tax/spring-tax.xml

```diff
@@ -33,4 +33,9 @@
 			</list>
 		</property>
 	</bean>
+
+	<bean id="taxParameterHelperDao" parent="platformAwareDao" class="edu.arizona.kfs.tax.dataaccess.impl.TaxParameterHelperDaoImpl" />
+
+	<bean id="taxParameterHelperService" class="edu.arizona.kfs.tax.service.impl.TaxParameterHelperServiceImpl" /> 
```

Comment:
Per research and GitHub discussion on 12/5, injecting service properties is a Kuali standard.